docs/standard/microservices-architecture/implement-resilient-applications/use-httpclientfactory-to-implement-resilient-http-requests.md

```diff
@@ -0,0 +1,163 @@
+---
+title: Use HttpClientFactory to implement resilient HTTP requests
+description: HttpClientFactory is an opinionated factory, available since .NET Core 2.1, for creating HttpClient instances to be used in your applications. 
+author: CESARDELATORRE
+ms.author: wiwagn
+ms.date: 06/08/2018
+---
+# Use HttpClientFactory to implement resilient HTTP requests
+
+HttpClientFactory is an opinionated factory, available since .NET Core 2.1, for creating HttpClient instances to be used in your applications. 
+
+## Issues with the original HttpClient class available in .NET Core
+
+The original and well-know [HttpClient](https://docs.microsoft.com/dotnet/api/system.net.http.httpclient?view=netstandard-2.0) class can be easily used, but in some cases, it is not being properly used by many developers. 
+
+As a first issue, while this class is disposable, using it with the using statement is not the best choice because even when you dispose HttpClient object, the underlying socket is not immediately released and can cause a serious issue named ‘sockets exhaustion’ as explained in this [blog post by Simon Timms](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/). 
```

Comment:
for blog posts, we usually refer to them by their titles, not by who wrote. Using title also helps with SEO. For example:
as explained in this [blog post by Simon Timms](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/) ->
For more information about X, see [You're using HttpClient wrong and it is destabilizing your software](https://aspnetmonsters.com/2016/08/2016-08-27-httpclientwrong/) blog post.

Reply:
👍